ml-agents/mlagents/trainers/policy/policy.py

```diff
@@ -29,14 +29,10 @@ def __init__(
         condition_sigma_on_obs: bool = True,
     ):
         self.behavior_spec = behavior_spec
+        self.action_spec = behavior_spec.action_spec
```

Comment:
Why do we need a reference to action_spec if we already have an access to the behavior_specs ? Seems unnecessary to me.